kubernetes/utils/create_from_yaml.py

```diff
@@ -23,51 +23,85 @@
 from kubernetes import client
 
 
-def create_from_yaml(k8s_client, yaml_file, verbose=False, **kwargs):
+def create_from_yaml(
+        k8s_client,
+        yaml_file,
+        verbose=False,
+        output_list=False,
+        **kwargs):
     """
     Perform an action from a yaml file. Pass True for verbose to
     print confirmation information.
     Input:
     yaml_file: string. Contains the path to yaml file.
     k8s_cline: an ApiClient object, initialized with the client args.
+    verbose: If True, print confirmation from the create action. Default is False.
+    output_list: compatibility option with v8.0.0. Default is False.
+        Function returns a single api object when there is only one when set False.
+        Does not affect when multiple objects are generated.
```

Comment:
I don't think it's a breaking change, because we didn't support list creation. 

I prefer not having the `output_list` parameter. Clients that create individual resources won't see difference, right?

Reply:
I think only if we have this right now the clients won’t see any difference. If we output lists all the time, we would have k8s_api returned in the current release and [k8s_api] with this PR if we are creating individual resource. I’m thinking to let the client choose what they want to do: if they want to create everything without checking the return type they can set ```output_list``` to ```True``` and just iterate the list. Otherwise if they want the legacy behavior they can choose not to pass any parameter.